src/CMakeLists.txt

```diff
@@ -1,8 +1,22 @@
-add_library(sqlpp-mysql
-		connection.cpp
-		prepared_statement.cpp
-		char_result.cpp
-		bind_result.cpp
-		detail/connection_handle.cpp)
+
+set(sqlpp_mysql_SRCs
```

Comment:
Rather than using a set Variable here, you can just use 

`target_sources(sqlpp-mysql PRIVATE connection.pp) ` 

after you declared the library (so in line 12 for instance), which is preferred as you can control the visibility.

Reply:
Here I decided to do it this way just to facilitate the readability of the code, I believe they are a little cleaner this way, if you still prefer I can change without problems.

Reply:
I don't mean to make it like before (i.e. putting it back in add_library), I agree that it's more readable in a separate line. But you have the same advantage as with target_sources. Additionally you can also specify visibility and spawn multiple calls across directories if you need that, which is very clean imo. 
https://cmake.org/cmake/help/latest/command/target_sources.html

I would suggest using it.

Reply:
It's Done!